src/components/NavBar.js

```diff
@@ -1,31 +1,63 @@
+import React, {useState} from "react";
+import {Link} from "react-router-dom";
 import { faCaretDown } from "@fortawesome/free-solid-svg-icons";
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
+import { faHammer } from "@fortawesome/free-solid-svg-icons";
+import { faTools } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
 import './NavBar.css';
 import OfekLogo from '../assets/images/OfekLogo.png';
 
-const NavBar = () => {
+const NavBar = props => {
+    
+
+    const toggleSideBar = () => {
+        props.MinMaxMain();
+        if(props.mini) document.getElementById("sidebar").style.width = "250px";
+        else document.getElementById("sidebar").style.width = "85px";
+    };
+
   return (
-    <Router>
-        <nav id="nav" className="nav" role="navigation">
-            <ul className="nav-links">
-                <li>
-                    <Link to="/" className="navItem"><img className="ofeklogo" src={OfekLogo} alt="ofek unit logo" /></Link>
-                </li>
-                <li>
-                    <Link to="/about" className="navItem">About us</Link>
-                </li>
-                <li>
-                    <Link to="/pybuild" className="navItem">PyBuild</Link>
-                </li>
-                <li>
-                    <Link to="/tools" className="navItem"><FontAwesomeIcon icon={faCaretDown} style={{marginRight: "10px"}} />Tools</Link>
-                </li>
-            </ul>
+      <div>
+        <nav id="sidebar" className="sidebar" role="navigation"
+             onMouseOver={() => toggleSideBar()} onMouseOut={() => toggleSideBar()}>
+            <Link to={'/'} > 
+                <img  src={OfekLogo} alt="ofek unit logo" className="logo-icon" />
+                <span className="logo-text">F15-HUB</span>
+            </Link>
+
+            <Link to={'/about'}>
+                <FontAwesomeIcon icon={faQuestionCircle} className="side-bar-icon"/>
+                <span className="icon-text">About</span>
+            </Link>
+            <Link to={'/pybuild'} >
+                <FontAwesomeIcon icon={faHammer} className="side-bar-icon"/>
+                <span className="icon-text">PyBuild</span>
+            </Link>
+            <nav>
+                <div className="dropdown-menu">
+                    <div className="dropdown-menu-item">
+                    <div className="dropdown-container">
+                    <FontAwesomeIcon icon={faTools} className="side-bar-icon" />
+                    <span className="icon-text">Tools</span>
+                    <FontAwesomeIcon icon={faCaretDown} style={{marginLeft: "10px"}}/> 
+                    </div>
+                        <div className="dropdown-menu">
+                            <div className="dropdown-menu-item">
+                                <Link to="/cmd-word"><span  className="tool-item">CMD Word</span></Link>
```

Comment:
Read about how to render an **array** of jsx components. :smiley
It's an awsome property of react, REALLY fucking useful and could save you a lot of time in the future